src/Components/Locations.js

```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
+import { useState } from "react";
+
+const Locations = () => {
+  const [locations, setLocations] = useState([]);
+  const [show, setShow] = useState("Show Locations");
+
+  const handleClick = async () => {
+    if (show === "Show Locations") {
+      try {
+        const res = await axios.get(
+          "https://ghibliapi.herokuapp.com/locations"
```

Comment:
We want to make as few axios calls as possible. Instead of making the call on the click, you should make it on mount and then toggle showing the list with the button.